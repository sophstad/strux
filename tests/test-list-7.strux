int main() {
	LinkedList::string:: l = new LinkedList::string::();
	l.add("S");
	l.add("H");
	l.add("O");
	l.add("W");
	l.show();
	return 0;
}