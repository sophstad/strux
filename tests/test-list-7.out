+-------------+  +------+  +-----+  +-----+  +------+
| HELLO WORLD |->| HAII |->| OIO |->| WWW |->| NULL |
+-------------+  +------+  +-----+  +-----+  +------+
0                1         2        3        <- Index
+---+  +---+  +------+
| 0 |->| 1 |->| NULL |
+---+  +---+  +------+
0      1      <- Index
